### PR TITLE
Remove prefixes from dependabot commit messages

### PR DESCRIPTION
**Pull request by steverep**
## Proposed change

The docs are not clear.. 😞   Hopefully this fixes the redundant prefixes.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -14,5 +14,3 @@ updates:
     open-pull-requests-limit: 5
     commit-message:
```
Comment (ludeeus) [inline .github/dependabot.yml:15]: IMO the whole `commit-message` section can be removed.

Comment (steverep) [reply .github/dependabot.yml:15]: I don't have strong feelings either way.  I just figured some way of separating dev from packaged dependencies would be nice.  Wish dependabot would just label it as such.

Comment (frenck) [reply .github/dependabot.yml:15]: Dependabot does add labels

Comment (steverep) [reply .github/dependabot.yml:15]: Not in a way that allows you to filter development tendencies from production ones. It only adds the "dependencies" label, and another label based on the package manager.

Comment (ludeeus) [reply .github/dependabot.yml:15]: I don't think that matter at all, all requirement changes needs to be validated the same way regardless if the are nested under "devRequirements" or not.

Comment (steverep) [reply .github/dependabot.yml:15]: Fair enough. I am away from my computer for a bit, so feel free to push the change to remove it all.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**